### PR TITLE
ocrvs-6678 fix advanced search result any status tag

### PR DESCRIPTION
**Pull request by anamulhaquemollah**

### CODE REVIEW

File: packages/client/src/search/advancedSearch/utils.ts [hunk 1]
```diff
@@ -158,7 +158,7 @@ export const transformAdvancedSearchLocalStateToStoreData = (
         : localState.registrationStatuses === 'IN_REVIEW'
         ? [RegStatus.WaitingValidation, RegStatus.Validated, RegStatus.Declared]
         : localState.registrationStatuses === 'ALL'
-        ? Object.values(RegStatus)
+        ? [RegStatus.AnyStatus]
```
Comment (Zangetsu101) [inline packages/client/src/search/advancedSearch/utils.ts:161]: @anamulhaquemollah This is probably not the right place as we don't want to change the actual "data" which would most likely break the advance search functionality, instead we just want to change the "ui/representational data". My guess would be something is wrong in the following component: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/views/AdvancedSearch/AdvancedSearchResult.tsx#L484
when we are converting the data to ui

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/search/advancedSearch/utils.ts [hunk 2]
```diff
@@ -549,6 +549,7 @@ const getLabelForRegistrationStatus = (
     ALL: [
       RegStatus.Archived,
       RegStatus.Certified,
+      RegStatus.CorrectionRequested,
       RegStatus.DeclarationUpdated,
       RegStatus.Declared,
       RegStatus.InProgress,
```
Comment (Zangetsu101) [inline packages/client/src/search/advancedSearch/utils.ts:555]: Could we just use all the `RegStatuses` like how we are doing [here](https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/search/advancedSearch/utils.ts#L161)? That way the same bug won't show up when we add a new `RegStatus` next time

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/search/advancedSearch/utils.ts (hunk 2 above)
Comment (anamulhaquemollah) [reply packages/client/src/search/advancedSearch/utils.ts:555]: updated code accordingly. hope no bug will arise if RegStatus updated in future.

**Review by Zangetsu101 — APPROVED** [review]

**Comment by Zangetsu101** [comment]
@anamulhaquemollah The changes look good now. Could you please resolve the conflicts and add an entry in CHANGELOG?

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]
The changes in the yarn.lock file doesn't look right. if you run `yarn` at the root of you project, these conflicts should get fixed by themselves

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**